src/App.js

```diff
@@ -1,13 +1,15 @@
 import "./App.css";
 import { Routes, Route } from "react-router-dom";
-import { Home } from "./pages/home/home";
+import { Home, Login, Signup } from "pages";
 
 function App() {
   return (
     <div className="App">
       <Routes>
         <Route path="/" element={<Home />} />
         <Route path="/home" element={<Home />} />
+        <Route path="/login" element={<Login />} />
+        <Route path="/signup" element={<Signup />} />
```

Comment:
hey you can add require auth also.

Reply:
plan is to add requireauth on archive and all those routes. User should be able to write a note without an account then while saving navigate to login screen.